docs/timezone.md

```diff
@@ -45,6 +48,47 @@ tz1.getTransitions(now).next().done;  // => true
 tz2.getTransitions(now).next().done;  // => false
 ```
 
+## Temporal.TimeZone.from(thing: string | object) : Temporal.TimeZone
+
+**Parameters:**
+- `thing` (string or object): A `Temporal.TimeZone` object or a string from which to create a `Temporal.TimeZone`.
+
+**Returns:** a new `Temporal.TimeZone` object.
+
+This static method creates a new time zone from another value.
+If the value is a string, it can be:
+- a string that is accepted by `new Temporal.TimeZone()`;
+- a string in the ISO 8601 format including a time zone offset part.
+Or, if the value is an object, it can be another `Temporal.TimeZone` object, which is returned directly.
```

Comment:
#232 calls for `from` to always return a new object. But if the other `from`s currently do direct returns, then :+1: to this for now and they can all be fixed together.

Reply:
They do, as far as I can tell.